x-pack/platform/packages/shared/onechat/onechat-common/chat/events.ts

```diff
@@ -166,6 +183,8 @@ export const isThinkingCompleteEvent = (
 export interface RoundCompleteEventData {
   /** round that was completed */
   round: ConversationRound;
+  /** if true, it means the round was resumed, so we need to replace the last one instead of happening a new round */
+  resumed?: boolean;
```

Comment:
Re-using the same `round_complete` event. I could have introduced a `round_resumed` instead, but this was easier and I think it doesn't change much